parachain/src/command.rs

```diff
@@ -45,6 +45,9 @@ fn load_spec(id: &str, para_id: ParaId) -> std::result::Result<Box<dyn sc_servic
         "rococo-local" => Ok(Box::new(chain_spec::rococo_local_testnet_config(para_id))),
         "westend" => Ok(Box::new(chain_spec::westend_testnet_config(para_id))),
         "kusama" => Ok(Box::new(chain_spec::kusama_mainnet_config(para_id))),
+        "kintsugi" => Ok(Box::new(chain_spec::ChainSpec::from_json_bytes(
+            &include_bytes!("../res/kintsugi.json")[..],
```

Comment:
Not ideal for the executable size.. Why do we need this?

Reply:
I think it will be marginal, this is to simplify starting a Kintsugi collator.

Reply:
I think it's like 1.3 mb, which is not nothing. But not the end of the world either

Reply:
Hmm good point, let's leave for now and let @nud3l review. FWIW cumulus does this: https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/src/command.rs#L84-L127

Reply:
Cumulus is not exactly keeping things lean, compiling cumulus takes forever haha

Reply:
To be clear, my main issue is not so much the size of the binary itself, it's that I have a slight concern for the compilation time, as linking times generally scale with the size of the binary. However, since the complexity of the code doesn't increase, probably it'll be ok?

Reply:
I think it's ok that way - even with increase of compile times having a single source of truth of the gensis state that is directly with the code, is a big improvement.